stdlib/public/CompatibilityOverride/CompatibilityOverrideConcurrency.def

```diff
@@ -241,7 +241,7 @@ OVERRIDE_TASK_GROUP(taskGroup_initialize, void,
                     SWIFT_EXPORT_FROM(swift_Concurrency), SWIFT_CC(swift),
                     swift::, (TaskGroup *group, const Metadata *T), (group, T))
 
-OVERRIDE_TASK_GROUP(taskGroup_attachChild, void,
+OVERRIDE_TASK_STATUS(taskGroup_attachChild, void,
```

Comment:
I _think_ this is fine but am not 100% sure, the overrides I think are for older deployments in theory... Let's double check with @DougGregor just to be sure if we can change this.

Reply:
Yeah, it's fine. It only affects what file stamps out the boilerplate here.